src/psyclone/tests/domain/lfric/dyn_meshes_test.py

```diff
@@ -84,5 +83,5 @@ def test_add_mesh_symbols():
     for tag in mesh_names:
         sym = sym_table.lookup(tag)
         assert isinstance(sym, DataSymbol)
-        assert isinstance(sym.datatype, DataTypeSymbol)
-        assert sym.datatype.name == "mesh_type"
+        assert isinstance(sym.datatype, UnsupportedFortranType)  # is a pointer
```

Comment:
Would be good to check partial_type if/when we have it.

Reply:
Well, this doesn't have one and it is not a pointer :) deleted the comment